java/src/jmri/jmrix/can/cbus/CbusPredefinedMeters.java

```diff
@@ -43,11 +48,16 @@ public CbusPredefinedMeters(CanSystemConnectionMemo memo) {
                 memo.getSystemPrefix() + "V" + "CBUSCurrentMeter",
                 Meter.Unit.Milli, 0, 65535.0, 1.0, updateTask);
         
+        currentMeterExtra = new DefaultMeter.DefaultCurrentMeter(
+                memo.getSystemPrefix() + "V2" + "CBUSCurrentMeter",
```

Comment:
"V" stands for "Variable" and is the type letter for AnalogIO, which Meter extends.
I think it would be better if the system name was:
`memo.getSystemPrefix() + "V" + "CBUSCurrentMeter2"`
or
`memo.getSystemPrefix() + "V" + "CBUSCurrentMeterExtra"`

Reply:
Fixed